shared/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.h

```diff
@@ -185,19 +185,21 @@ namespace datadog::shared::nativeloader
         USHORT minor_version;
         USHORT build_version;
         USHORT qfe_version;
+        std::string inferred_version = "";
 
         RuntimeInformation() :
             runtime_type((COR_PRF_RUNTIME_TYPE) 0x0), major_version(0), minor_version(0), build_version(0), qfe_version(0)
         {
         }
 
         RuntimeInformation(COR_PRF_RUNTIME_TYPE runtime_type, USHORT major_version, USHORT minor_version,
-                           USHORT build_version, USHORT qfe_version) :
+                           USHORT build_version, USHORT qfe_version, const std::string inferred_version) :
```

Comment:
Nit: `const` here is not necessary, since it's a copy.

Reply:
Ah, _should_ it be a copy? That may be my mistake 😅